api/users/schemas.py

```diff
@@ -0,0 +1,8 @@
+import os
+import json
+
+here = os.path.split(os.path.abspath(__file__))[0]
+
+def from_json(fname):
```

Comment:
Nice 👍

Reply:
I like the use of the helper function. One nitpick: it's a bit of a misnomer. From the name, I'd expect `from_json` to take a JSON string as input, but it actually takes a filepath. Perhaps `load_jsonschema` would be more appropriate?

Also see my other comment about co-locating all the jsonschema-related code.